dev.sh

```diff
@@ -4,6 +4,13 @@ VENV=./venv
 
 python -m virtualenv $VENV --always-copy
 . $VENV/bin/activate
+pip_version=$(pip -V | cut -d' ' -f 2 | cut -d'.' -f 1)
+if [ "$pip_version" -lt 6 ];
```

Comment:
I checked it for version 6.0 it worked.
![screenshot-1](https://cloud.githubusercontent.com/assets/8709120/13258861/e69fb794-da7a-11e5-9178-7d5af31a18e7.jpg)
Only downgrade option available less than 6.0 was 1.5.6, it fails for this version.
Anyways upgrading pip unconditionally in dev script.